client/client/main.go

```diff
@@ -140,12 +162,15 @@ func main() {
 	reader := bufio.NewReader(os.Stdin)
 
 	for {
-		fmt.Print("Please enter word to search for (enter to exit): ")
-		keyword, _ := reader.ReadString('\n')
-		keyword = keyword[:len(keyword)-1]
-		if keyword == "" {
+		fmt.Print("Please enter words to search for separated by spaces (enter to exit): ")
+		input, _ := reader.ReadString('\n')
+		input = strings.TrimRight(input, "\n")
+		if input == "" {
 			break
 		}
-		performSearchWord(cli, keyword)
+		keywords := strings.Split(input, " ")
+		for _, keyword := range keywords {
+			performSearchWord(cli, keyword)
```

Comment:
The current output looks like this:
```
Please enter words to search for separated by spaces (enter to exit): happy friends non-existing
Files containing the word "happy":
	/keybase/private/jiaxin/happy.txt
	/keybase/private/jiaxin/what/happy.txt

Files containing the word "friends":
	/keybase/private/jiaxin/happy.txt
	/keybase/private/jiaxin/unhappy.txt

No file contains the word "non-existing".
```

It's more like a shorthand so that you can concatenate multiple search queries with spaces to run them one by one.